errors/errors_task.go

```diff
@@ -130,3 +130,21 @@ func (err *TaskCancelledNoTerminalError) Error() string {
 func (err *TaskCancelledNoTerminalError) Code() int {
 	return CodeTaskCancelled
 }
+
+// TaskCancelledNoTerminalError is returned when trying to run a task with a prompt in a non-terminal environment.
```

Comment:
Looks good. Minor thing, but this comment should probably be updated.

Reply:
DOH! Copy and paste snafu